README.md

```diff
@@ -1,32 +1,62 @@
 # Eclipse Buildship: Gradle integration for the Eclipse IDE <a href="https://builds.gradle.org/viewType.html?buildTypeId=Tooling_Buildship_Checkpoint_Full_Test_Coverage_Phase_33"><img src="https://builds.gradle.org/guestAuth/app/rest/builds/buildType:(id:Tooling_Buildship_Checkpoint_Full_Test_Coverage_Phase_33)/statusIcon"/></a>
 
+## Getting Started: Declarative Support
+
+> [!IMPORTANT]
+> This is a forked repository that is adding [Declarative Gradle](https://declarative.gradle.org/) support to normal Buildship.
+> The following instructions will help you to get started.
+
+### Setup: update site-based installation
+
+> [!TIP]
+> This installation method is preferred if you don't want to develop the plugins.
+
+1. Open the "Help > Install New Software..." dialog
+2. In the "Work with:" field, paste the following link: https://hegyibalint.github.io/buildship/
+3. Under "Buildship Extras - Incubating" there should be a new option: _"Buildship - Gradle Declarative editor support"_
+4. Select the option and proceed with the installation
+5. After installation, go to the "Settings > Gradle > Experimental features" menu, and select the `lsp-all.jar` file (the file is the result of `shadowJar` assembled by [declarative-lsp](https://github.com/gradle/declarative-lsp), see that repository for further instructions)
+
+The installation can be kept up-to-date, just go to "Help > Check for Updates".
+If there is a new snapshot version of the plugin, the dialog will offer the new version.
```

Comment:
:x: This needs to go, I will delete it

Reply:
Just add a line here and reference a dedicated document in the `docs` directory.

Reply:
Moved to `docs/`

Reply:
Pushed a single-line TIP admonition about where to find the docs